client/src/components/new-request/name-input.vue

```diff
@@ -9,10 +9,21 @@
                   :label="nameLabel"
                   v-model="nameSearch">
       <template v-slot:append>
-        <v-icon class="name-search-icon"
-                id="name-input-icon"
-                color="primary"
-                @click="startAnalyzeName">mdi-magnify</v-icon>
+        <v-tooltip bottom
+                   content-class="bottom-tooltip search-tooltip"
+                   transition="fade-transition"
+                   :disabled="!isSearchAgain">
+          <template v-slot:activator="scope">
+            <v-icon class="name-search-icon"
+                    id="name-input-icon"
+                    color="primary"
+                    v-on="scope.on"
+                    @click="startAnalyzeName">mdi-magnify</v-icon>
+          </template>
+          <template>
```

Comment:
Since you're not specifying which slot this template is for, do you need it?

Reply:
Same for the other tooltips.

Reply:
Indeed we do, this is the template for the ToolTip content.

Reply:
OK, the default template I guess. I just learned something! LOL

Reply:
I see what you mean, in this case we can do without!